webdriver/src/com/blazemeter/jmeter/webdriver/phantomjs/PhantomJSDriverConfig.java

```diff
@@ -34,6 +36,13 @@ protected Capabilities createCapabilities() {
         capabilities.setCapability(CapabilityType.PROXY, createProxy());
         capabilities.setCapability(PhantomJSDriverService.PHANTOMJS_EXECUTABLE_PATH_PROPERTY,
                 getPhantomJsExecutablePath());
+        String cliArgs = JMeterContextService.getContext().getVariables().get("phantomjs.cli.args");

```

Comment:
Please pull from master branch, there were another PR relating to this, which solves CLI args case in a better, more jmeter-standard way.